addons/web_tour/static/src/js/tip.js

```diff
@@ -315,8 +315,29 @@ var Tip = Widget.extend({
                 of: this.$anchor,
                 collision: "none",
                 using: props => {
-                    this.el.style.setProperty('top', `${props.top}px`, 'important');
-                    this.el.style.setProperty('left', `${props.left}px`, 'important');
+                    const {top} = props;
+                    let {left} = props;
+                    const hasInnerText = !!this.$anchor.contents().first().text().trim().length;
```

Comment:
As you only use this value in your if statement in the next line and the if works with falsy and truthy values, I think the double logical not is useless in this case. :wink: